images/win/scripts/Installers/Validate-R.ps1

```diff
@@ -0,0 +1,15 @@
+################################################################################
+##  File:  Validate-R.ps1
+##  Desc:  Validate R.
+################################################################################
+
+# Validate Rscript
+if (Get-Command -Name 'Rscript')
+{
+    Write-Host "Rscript on path"
+}
+else
+{
+    Write-Host 'Rscript is not on path'
+    exit 1
+}
```

Comment:
Could you please implement new test for this tool?
See https://github.com/actions/virtual-environments/pull/1205

Reply:
Done.